react/features/device-selection/functions.js

```diff
@@ -62,6 +64,8 @@ export function getDeviceSelectionDialogProps(stateful: Object | Function) {
             !JitsiMeetJS.isCollectingLocalStats(),
         hideAudioOutputSelect: !JitsiMeetJS.mediaDevices
                             .isDeviceChangeAvailable('output'),
+        hideVideoInputPreview: isMobileSafari,
```

Comment:
I thought of using a single prop here, but all of the others were more verbose. WDYT?